FlintCore/Actions/StaticActionBinding.swift

```diff
@@ -152,11 +152,11 @@ extension StaticActionBinding where ActionType.InputType == NoInput {
 
 /// Overloads for the case where there is neither a presenter nor an input
 extension StaticActionBinding where ActionType.InputType == NoInput, ActionType.PresenterType == NoPresenter {
-    public func perform(completion: ((ActionOutcome) -> ())? = nil) {
+    public func perform(withCompletion completion: ((ActionOutcome) -> ())? = nil) {
         session.perform(self, input: .noInput, presenter: NoPresenter(), completion: completion)
     }
 
-    public func perform(userInitiated: Bool,
+    public func perform(withUserInitiated userInitiated: Bool,
```

Comment:
It doesn’t read well when you have a bool as the first one. I’d suggest putting this parameter second and making “withSource” first. If that doesn’t work for you, I’d just remove with I guess.